.analysis_options_repo

```diff
@@ -67,35 +69,41 @@ linter:
     - always_specify_types
     - annotate_overrides
     - avoid_as
+    # - avoid_function_literals_in_foreach_calls # not yet tested
     - avoid_init_to_null
     - avoid_return_types_on_setters
     - await_only_futures
     - camel_case_types
+    # - cascade_invocations # not yet tested
     # - constant_identifier_names # https://github.com/dart-lang/linter/issues/204
     - directives_ordering
     - empty_catches
     - empty_constructor_bodies
     - implementation_imports
     - library_names
     - library_prefixes
-    - no_adjacent_strings_in_list
-    - no_duplicate_case_values
     - non_constant_identifier_names
+    # - omit_local_variable_types # opposite of always_specify_types
     # - one_member_abstracts # too many false positives
     # - only_throw_errors # https://github.com/flutter/flutter/issues/5792
-    # - overridden_fields
+    # - overridden_fields # not yet tested
     - package_api_docs
     - package_prefixed_library_names
     # - parameter_assignments # we do this commonly
-    # - prefer_final_fields # https://github.com/dart-lang/sdk/issues/29197
-    - prefer_collection_literal
+    # - prefer_adjacent_string_concatenation # not yet tested
+    - prefer_collection_literals
     - prefer_const_constructors
     - prefer_contains
+    # - prefer_expression_function_bodies # not yet tested
+    # - prefer_final_fields # https://github.com/dart-lang/linter/issues/506
     - prefer_final_locals
+    # - prefer_function_declarations_over_variables # not yet tested
     - prefer_initializing_formals
+    # - prefer_interpolation_to_compose_strings # not yet tested
     - prefer_is_empty
     - prefer_is_not_empty
-    - public_member_api_docs # this is the only difference from .analysis_options
+    # - public_member_api_docs # this is the only difference from .analysis_options_repo
```

Comment:
I missed this when reviewing it (and will fix it momentarily), but just FYI, this specific line broke `--dartdocs` in `flutter analyze`. :-)